.prettierignore

```diff
@@ -0,0 +1,22 @@
+# In addition to gitignore...
+
+# generated files
+pnpm-lock.yaml
+
+# we disagree with prettier and we are even more opinionated than they are
+*.hbs
+*.html
+*.md
+
+# these files would be YUGE if we prettified them
+MOCK_DATA.json
+tests/ember-data__request/.mock-cache/
```

Comment:
I would more broadly ignore any `.mock-cache` directory, there's soon to be ~10 of them